api/view/static/openapi.json

```diff
@@ -0,0 +1,298 @@
+            "required": false,
+            "schema": {
+              "type": "string",
+              "enum": [
+                "geometryview",
+                "centroid",
+                "simplifiedgeom",
+                "alternates"
+              ]
+            }
+          },
+          {
+            "name": "_format",
+            "in": "query",
+            "description": "Media type. If none selected, defaults to 'text/html' or whatever is in the Accept header.",
+            "required": false,
+            "schema": {
+              "type": "string",
+              "enum": [
+                "text/html",
+                "application/json",
+                "text/turtle",
+                "text/plain"
+              ]
+            }
+          }
+        ]
+      }
+    },
+    "/search/latlng/{x},{y}": {
+      "get": {
+        "tags": [
+          "search"
+        ],
+        "summary": "Get geometry instances that contain specified position",
+        "description": "",
+        "responses": {
+          "200": {
+            "description": "OK",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "type": "string"
+                }
+              },
+              "text/html": {
+                "schema": {
+                  "type": "string"
+                }
+              }
+            }
+          }
+        },
+        "operationId": "searchGeomByPosition",
+        "parameters": [
+          {
+            "name": "x",
+            "in": "path",
+            "description": "x",
+            "required": true,
+            "schema": {
+              "type": "number",
+              "example": 142.111
+            }
+          },
+          {
+            "name": "y",
+            "in": "path",
+            "description": "y",
+            "required": true,
+            "schema": {
+              "type": "number",
+              "example": -32.1231
+            }
+          },
+          {
+            "name": "crs",
+            "in": "query",
+            "description": "crs",
+            "required": false,
+            "schema": {
+              "type": "number"
+            }
+          }
+        ]
+      }
+    },
+    "/search/latlng/{x},{y}/dataset/{dataset_id}": {
+      "get": {
+        "tags": [
+          "search"
+        ],
+        "summary": "Get geometry instances that contain specified position for a specified dataset",
+        "description": "",
+        "responses": {
+          "200": {
+            "description": "OK",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "type": "string"
+                }
+              },
+              "text/html": {
+                "schema": {
+                  "type": "string"
+                }
+              }
+            }
+          }
+        },
+        "operationId": "searchGeomByLatLngAndDatasetId",
```

Comment:
searchGeomByPositionAndDataset

Reply:
... where 'dataset' is some kind of collection? 

OGC has struggled for years with the notion of 'layer' which was introduced by WMS and matched the old light-table concept. But when they moved on to WFS and SOS they realised that some kind of subsetting was required, though the basis for subsetting was various. In some cases it was a single feature-type. In some cases it was a group of types from a specified theme (e.g. hydrography). In some cases it is custodianship. In SOS cases it is some grouping of FoI, OP, sensor, maybe time interval. Conclusion: there are many equally legitimate ways to create collections. 'dataset' is probably a good enough name, but always need to be a bit aware.

Reply:
But in order for this to be coherent, need a 'GetDatasetIDs' operation ...

Reply:
i'll add that in the spec. will create a separate issue for implementation

Reply:
how about `/dataset/` ? This returns list of dataset items, with id, and potentially other info...

Reply:
+1